docs/basic-remote-setup.md

```diff
@@ -53,16 +56,18 @@ You need to configure `your-host` in some services to provide the needed public
 
 ```bash
 PROXY_HTTP_ADDR=0.0.0.0:9200 \
-KONNECTD_ISS=https://your-host:9200 \
-REVA_OIDC_ISSUER=https://your-host:9200 \
-PHOENIX_OIDC_AUTHORITY=https://your-host:9200 \
-PHOENIX_WEB_CONFIG_SERVER=https://your-host:9200 \
-PHOENIX_OIDC_METADATA_URL=https://your-host:9200/.well-known/openid-configuration \
+KONNECTD_ISS=https://your-server:9200 \
+REVA_OIDC_ISSUER=https://your-server:9200 \
+PHOENIX_OIDC_AUTHORITY=https://your-server:9200 \
+PHOENIX_WEB_CONFIG_SERVER=https://your-server:9200 \
+PHOENIX_OIDC_METADATA_URL=https://your-server:9200/.well-known/openid-configuration \
 REVA_DATAGATEWAY_URL=https://your-server:9200/data \
 REVA_FRONTEND_URL=https://your-server:9200 \
 PROXY_TRANSPORT_TLS_KEY=./certs/your-host.key \
 PROXY_TRANSPORT_TLS_CERT=./certs/your-host.crt \
 KONNECTD_TLS=0 \
+REVA_DATAGATEWAY_URL=https://your-server:9200/data \
```

Comment:
`REVA_DATAGATEWAY_URL` and `REVA_FRONTEND_URL` already exist a few lines above. We already merged a PR with those two lines a few days ago. If you want to have them in this order, please remove them above so that we don't duplicate the env vars.

Reply:
uups :-)